set.mm

```diff
@@ -172276,8 +172282,8 @@ seq m ( x. , G ) ~~> z ) ) $.
     HMWBVMUNRUOQVFVKVFVKVFVJWEUPNUQVFVJWDVFSVJURTZSVJUSTZVFVIPCWFWAVIUTRVFSPCVJ
     PCWFWGVAVBWDSVJVCOVDVEQ $.
 
-  $( A number is real iff its imaginary exponential has absolute value one.
-     (Contributed by NM, 21-Aug-2008.) $)
+  $( A complex number is real iff the exponential of its imaginary part has
```

Comment:
Hmm, ` _i x. A ` is not the imaginary part of ` A ` . (that would be https://us.metamath.org/mpeuni/df-im.html ).

A verbose wording would be "the exponential of the number times `_i`" and I suppose that's probably better than "its imaginary exponential" (although I'd also be OK with keeping the latter even though it is not really self explanatory).

Reply:
You're absolutely right, I overlooked it. I chose `the exponential of its product with _i has ...`.

On the other hand, the function $t \mapsto e^{it}$ appears often enough that it may be called "imaginary exponential" ? I don't know how standard that is.

Reply:
If it is standard I didn't find it in a (admittedly cursory) internet search.